Engine/csound_orc_semantics.c

```diff
@@ -1044,22 +1029,22 @@ int32_t check_out_args(CSOUND* csound, char* outArgsFound, char* opOutArgs)
   }
 
   {
-    int32_t argsFoundCount = argsRequired(outArgsFound);
-    int32_t argsRequiredCount = argsRequired(opOutArgs);
-    char** argsRequired = splitArgs(csound, opOutArgs);
+    int32_t argsFoundCount = args_required(outArgsFound);
+    int32_t args_requiredCount = args_required(opOutArgs);
+    char** args_required = split_args(csound, opOutArgs);
```

Comment:
Naming this the same as the args_required() function is a little confusing. (Understanding that this was a pre-existing problem here and in check_in_args.) Perhaps we can use this opportunity to choose a different name.

Reply:
that is fixed now.